pkg/cloud/cloud.go

```diff
@@ -710,11 +731,20 @@ func (c *cloud) WaitForAttachmentState(ctx context.Context, volumeID, expectedSt
 			return true, nil
 		}
 		// continue waiting
-		klog.V(2).InfoS("Waiting for volume state", "volumeID", volumeID, "actual", attachmentState, "desired", expectedState)
+		klog.V(4).InfoS("Waiting for volume state", "volumeID", volumeID, "actual", attachmentState, "desired", expectedState)
```

Comment:
Why? This log message has been historically very useful for debugging attach-related issues

Reply:
Higher verbosity levels are typically used for this category of logs which are very frequent, logging volume state results in a high volume of log entries. 

Instead, we now log the actual attachment / detachment times which also provides useful information for debugging, a more comprehensive view of a volume's attachment/detachment lifecycle, and substantially reduces the overall volume of log entries at the default verbosity level.